modules/play-json/src/test/scala/com/evolutiongaming/skafka/producer/JsonProducerSpec.scala

```diff
@@ -6,32 +6,32 @@ import com.evolutiongaming.skafka.{Bytes, Partition, ToBytes, TopicPartition}
 import play.api.libs.json.{JsString, Json}
 import org.scalatest.funsuite.AnyFunSuite
 import org.scalatest.matchers.should.Matchers
+import scala.util.Try
 
 class JsonProducerSpec extends AnyFunSuite with Matchers {
-
   test("apply") {
     val metadata = RecordMetadata(TopicPartition("topic", Partition.min))
     var actual = Option.empty[(Option[Bytes], Option[Bytes])]
-    type Id[A] = A
-    val send = new Producer.Send[Id] {
+
+    val send = new Producer.Send[Try] {
       def apply[K, V](
         record: ProducerRecord[K, V])(implicit
-        toBytesK: ToBytes[Id, K],
-        toBytesV: ToBytes[Id, V]
-      ) = {
+        toBytesK: ToBytes[Try, K],
+        toBytesV: ToBytes[Try, V]
+      ): Try[Try[RecordMetadata]] = {
```

Comment:
I was getting `scala cyclic aliasing or subtyping involving type Id` with Scala 2.13 under `source:3` here so I decided to change it to simplest concrete type that supports `FromTry`.

Reply:
could it work setting types explicitly without asking compiler to derive them?

Reply:
Now experimenting with it a little bit more... 
It breaks when the return type is set to `Id[Id[RecordMetadata]]`, but it actually works fine with `RecordMetadata` 😄

Reply:
cool